onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -238,7 +238,7 @@ def aten_addcmul(
     return op.Add(self, op.Mul(op.Mul(value, tensor1), tensor2))
 
 
-@torch_op("aten::addmm")
+@torch_op("aten::addmm", traceable=True)
```

Comment:
For now, we can make it trace only and explicitly cast beta and alpha with

alpha=float(alpha), beta=float(beta)